src/env/env.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, expect, it, afterEach } from 'vitest';
+
+describe('env', () => {
+  afterEach(() => {
+    delete global.window;
```

Comment:
i think we should use globalThis rather than global because it has interop between browser and node
Probably doesnt matter for this test because I assume this test will not run in the browser in the future?
But just to get used to globalThis over global

Reply:
yes i think yes, this global interaction i also think it broke several things